Riot/Modules/Spaces/SpaceRoomList/ExploreRoomCoordinator.swift

```diff
@@ -248,3 +282,93 @@ extension ExploreRoomCoordinator: UIAdaptivePresentationControllerDelegate {
     }
     
 }
+
+extension ExploreRoomCoordinator: RoomViewControllerDelegate {
```

Comment:
I think the real question here is whether we can compose this together with the existing RoomCoordinator to not have to duplicate all this functionality? I'm sure it's possible, if not, then we should change the RoomCoordinator so that it is.

Reply:
This issue is reported here: https://github.com/vector-im/element-ios/issues/5710